app/controllers/application_controller.rb

```diff
@@ -1,3 +1,10 @@
 class ApplicationController < ActionController::Base
   before_action :authenticate_user!
+  before_action :ensure_signup_complete, only: %i[new create update destroy]
```

Comment:
why only has these actions? what about others? maybe it's better to remove this only section

Reply:
Without this - error: [https://yadi.sk/i/H3h4i86L3aTEJA](https://yadi.sk/i/H3h4i86L3aTEJA)

Reply:
I mean that we can have other actions and this will fail on some rent action, if you check the repo code